redaxo/src/core/default.config.yml

```diff
@@ -7,6 +7,21 @@ error_email: null
 fileperm: '0664'
 dirperm: '0775'
 session_duration: 7200
+session:
+    backend:
+        cookie:
+            lifetime: null
```

Comment:
null in der config bedeutet dass das settings nicht angefasst wird.
ich habe hier alle "möglichen keys" gelistet, sodass man es leichter hat die werte zu ändern - wenn gewünscht.